nodemon.json

```diff
@@ -1,5 +1,4 @@
 {
-  "watch": ["dist/"],
-  "ext": "js",
-  "delay": 500
-}
\ No newline at end of file
+  "watch": ["src/"],
```

Comment:
why you change the build directory? @dicka88

Reply:
since using node-ts, we should not build the code first to run the project, so `nodemon` only watches the /src directory that contains the typescript code